frontend/src/entry/model/InferenceServices.tsx

```diff
@@ -27,6 +55,41 @@ export default function InferenceServices({ model }: InferenceProps) {
     [model, inferences],
   )
 
+  const handleCreateToken = async () => {
+    setErrorMessage('')
+    const error = MultipleErrorWrapper('Unable to create an authenication token', { isTokensError, isUiConfigError })
+    if (error) return error
+    const authorizationTokenName = uiConfig?.inference.authorizationTokenName
+    const authorizationAccessKeys = tokens.filter((value) => value.description === authorizationTokenName)
+    if (authorizationTokenName && authorizationAccessKeys) {
```

Comment:
`authorizationAccessKeys` will always be an array so this part of the check will always be true. Either check the length `authorizationAccessKeys.length > 0` or remove it from the check, depending on whether or not it is required.